mesonbuild/interpreter/interpreter.py

```diff
@@ -2124,6 +2124,8 @@ def func_run_target(self, node: mparser.FunctionNode, args: T.Tuple[str],
     def func_alias_target(self, node: mparser.BaseNode, args: T.Tuple[str, T.List[build.Target]],
                           kwargs: 'TYPE_kwargs') -> build.AliasTarget:
         name, deps = args
+        if any(isinstance(d, build.RunTarget) for d in deps):
+            FeatureNew.single_use('Allow alias_target to comprise run_targets', '0.60.0', self.subproject)
```

Comment:
"allow" is unusual wording compared to most FeatureNew cases.

Reply:
Open to suggestions.